### ISSUE TITLE
[4.8.0] Infill going through the wall

### ISSUE THREAD

**Issue by yngndrw** [Ultimaker/Cura#9460]
### Application Version

4.8.0

### Platform

Windows 10

### Printer

E3D Toolchanger

### Reproduction steps

I just loaded my model with my standard settings.

### Actual results

On layer #11, the infill has been placed outside of the model.

### Expected results

The infill should be constrained to the model.

A picture is a thousand words:
![image](https://user-images.githubusercontent.com/1834655/112739706-66467480-8f6e-11eb-98cd-511e1bd28594.png)

I haven't finished this design or configured the correct settings, I was just looking to see approximately how long it would take to print when I noticed this issue. The model was made using Onshape.

Project file and .stl:
[Mount Base - Test.zip](https://github.com/Ultimaker/Cura/files/6216706/Mount.Base.-.Test.zip)

[cura.log](https://github.com/Ultimaker/Cura/files/6216707/cura.log)

**Comment by fvrmr** [comment]
Hi @yngndrw thank you for your bug report. 
I can reproduce this in 4.8 but in our Arachne beta release it doesn't happen. Could you check if there the issue doesn't occur either?
![Screenshot (139)](https://user-images.githubusercontent.com/77000924/112822323-178cfd80-9088-11eb-8576-f8a54a473bce.png)

You can check the Arachne release here: 
https://github.com/Ultimaker/Cura/releases/tag/Arachne_engine_beta

Also I am wondering why you want to print this at `one at a time`? Because you only have one model. 
I will discuss the issue with the team since it is happening on 4.8 
Keep you posted.

**Comment by yngndrw** [comment]
Arachne does seem to work correctly, although I would note that it seems to be approx 10-20% slower at slicing. (Non-scientific measurement) The supports are slightly different, especially the little extra bits that tree supports like to add, there's a few single-layer rings that it has placed around the horizontal holes in the lower portion - Although this isn't unique to Arachne)

The Cura profile is just my main PETG profile with some changes from the last thing I printed. I've started using `one at a time` in order to reduce stringing between different parts, although most of the time I just print a single thing.
